azure/terraform/main.tf

```diff
@@ -40,6 +52,17 @@ provider "azurerm" {
   features {}
 }
 
+provider "helm" {
+  kubernetes {
+    config_path = local_file.helm_kubeconfig.filename
```

Comment:
I believe there is a slightly more elegant way to initialize helm provider without creating an intermediate kubeconfig file like described here https://myownpicloud.wordpress.com/2021/04/20/building-aks-cluster-with-terraform-and-helm-providers/ . At least that's what I usually do for AWS

Reply:
Done.